.chloggen/jaegerremotesampling_replace_thrift-gen_with_proto-gen.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: jaegerremotesampling
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Replace Thrift-gen with Proto-gen types for sampling strategies
```

Comment:
What do you see as the user facing change?

Reply:
The grpc interface will change. The returned values do not match.